AppLovinMAXDemo/Plugins/AppLovinMAX/Source/AppLovinMAX/AppLovinMAX_UPL_IOS.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- AppLovin plugin additions -->
+<root>
+  <init>
+    <log text="AppLovin IOS SDK init"/>
```

Comment:
When does this get executed? I'm wondering if we can use this for dependency management.

Reply:
During compilation. Apart from a place to add custom Info.plist values, we don't seem to get further options for iOS, but could do some file copying/management ([UPL docs](https://docs.unrealengine.com/4.26/en-US/SharingAndReleasing/Mobile/UnrealPluginLanguage/))